config/routes.rb

```diff
@@ -99,6 +99,7 @@
         get :for_order_cycle
         get :visible
         post :bulk_update, as: :bulk_update
+        post :invite_manager
```

Comment:
by the look of this `resource` declaration you can clearly see that `enterprises` is getting too long. We're coupling way too many things with that resource, and we're already aware of that.